guides/v2.2/frontend-dev-guide/css-topics/css-overview.md

```diff
@@ -44,7 +44,7 @@ In the Blank theme, the buttons of the `.action.primary` class, so called *prima
 OrangeCo wants to change the color of the primary buttons to orange. To achieve this, they do the following:
 
 1. Create a new Orange theme, which inherits from the Blank [theme](https://glossary.magento.com/theme).
-2. In the Orange theme directory add the overriding `app/design/frontend/OrangeCo/orange/web/css/source/_theme.less` file with the following code:
+1. In the Orange theme directory add the overriding `app/design/frontend/OrangeCo/orange/web/css/source/_theme.less` file with the following code:
```

Comment:
@dobooth Why was the 2nd point made `1` as well? Is this a typo? It's not present in my commit.

Reply:
In markdown, ordered lists need only the number 1. In fact, if in the future we need to insert a line, we don't have to renumber ever other line after. Therefore, all of our ordered list lines begin with 1.

Reply:
I understand @hguthrie, but what's the need to replace the following
`1. Sample`
`2. Sample`
**with**
`1. Sample`
`1. Sample` ?